recipes/libhwloc/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "libhwloc" %}
+{% set pname = "hwloc" %}
+{% set version = "1.11.4" %}
+{% set sha256 = "1b6a58049c31ce36aff162cf4332998fd468486bd08fdfe0249a47437311512d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ pname }}-{{ version }}.tar.gz
+  url: https://www.open-mpi.org/software/hwloc/v1.11/downloads/{{ pname }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - disable-failing-test.patch
```

Comment:
Can you just add a comment about the patch to help us remember the reasons for it?